test/new-e2e/pkg/e2e/suite.go

```diff
@@ -283,7 +283,7 @@ func (bs *BaseSuite[Env]) reconcileEnv(targetProvisioners ProvisionerMap) {
 		}
 
 		if err != nil {
-			panic(fmt.Errorf("unable to provision stack: %s, provisioner %s, err: %v", bs.params.stackName, id, err))
+			panic(fmt.Errorf("your stack '%s' provisioning failed, check logs above. Provisioner was %s, failed with err: %v", bs.params.stackName, id, err))
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
If we panic here, doesn't this prevent cleaning up a failing deployment ?

Reply:
What if you return the error and let the caller decide how to handle it ?

Reply:
Yep that's what I pushed, but clean implementation requires larger changes, so we're back at provisioning in `BeforeTest`.